### ISSUE TITLE
Conservation of mass

### ISSUE THREAD

**Issue by sonjageorgievska** [sonjageorgievska/Arena#6]

**Comment by sonjageorgievska** [comment]
@philiprn The size of the kernel when smoothing a MAC in this case should actually depend on the current estimation of crowd density in the bin of the MAC, right? If the crowd is very dense, we cannot assume Brownian motion with 1m/s speed. Apropos the paper on Probing crowd density, where they have made research on how speed depends on density.

**Comment by philiprn** [comment]
Yes, the Brownian motion should be related to the local density using a so-called fundamental diagram.

**Comment by philiprn** [comment]
Smoothing a density distribution with a kernel that adapts to the local density values seems complicated.
Maybe we can subdivide the distribution in sectors and smooth each sector separately using the average crowd density value of the sector.

**Comment by philiprn** [comment]
Maybe we should assume a minimal pedestrian velocity, instead of a maximum velocity as a function of density, because it is unlikely that people will walk fast with a specific target in mind. People will also not walk collectively in all directions in the sense of sudden collective dispersion, which is the concept of Brownian diffusion. If they walk, they walk very slowly.
An average minimum velocity is ~0.5 m/s (see Willis et al. 2004).

**Comment by sonjageorgievska** [comment]
I have to take a look at the papers, because the minimal velocity is 0 by common sense :), if everybody is standing still and watching the show. Or if the crowd is too dense so that they cannot move. 

Anyway, we already talked that we will limit ourselves by the maximal velocity, but that means that anything in [0, max_velocity] is acceptable. We need the ground truth to finalize this issue :(
